packages/orm/src/Model.ts

```diff
@@ -79,12 +134,12 @@ export default class Model {
    * Don't use this to create a new record; Use the static method {@link Model#create}.
    * @param  {Object} props - the properties to instantiate with
    */
-  constructor(props?: object) {
+  constructor(props?: Attrs) {
```

Comment:
Implicitly checks if a derived class is instantiated with a proper object. The constructor then uses `_initFields` to define the class properties.